logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/receiver/conversation/message/NewMessageEventHandler.kt

```diff
@@ -62,19 +62,7 @@ internal class NewMessageEventHandlerImpl(
 
     override suspend fun handleNewMLSMessage(event: Event.Conversation.NewMLSMessage) {
         mlsMessageUnpacker.unpackMlsMessage(event)
-            .onFailure {
-                applicationMessageHandler.handleDecryptionError(
-                    eventId = event.id,
-                    conversationId = event.conversationId,
-                    timestampIso = event.timestampIso,
-                    senderUserId = event.senderUserId,
-                    senderClientId = ClientId(""), // TODO(mls): client ID not available for MLS messages
-                    content = MessageContent.FailedDecryption(
-                        isDecryptionResolved = false,
-                        senderUserId = event.senderUserId
-                    )
-                )
-            }.onSuccess {
+            .onSuccess {
```

Comment:
Since it's temporary, what do you think of having a todo to remind us for covering later?

Reply:
Sure